### PR TITLE
DockMod 3 for Yosemite

### PR DESCRIPTION
**Pull request by RyuX51**
DockMod 3 is a dock customiser for 10.10 Yosemite. For 10.9 Mavericks and lower please use cask dockmod (DockMod 2) instead.

### CODE REVIEW

File: Casks/dockmod3.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+cask :v1 => 'dockmod3' do
+  version '3.01'
+  sha256 '1d7582c5ec59dd4dc15a07db10042b8654ff0047091bb401e98a4bd6810699bb'
+
+  url "http://spyresoft.com/dockmod/download.php?version=#{version}"
+  appcast 'http://spyresoft.com/dockmod/updates3.xml',
+          sha256 => :no_check
```
Comment (vitorgalvao) [inline Casks/dockmod3.rb]: This must be set, it can’t be left like this.

**Comment by vitorgalvao** [comment]
Thank you for the contribution, but you’re going about this the wrong way. Either the current cask will be updated, of it’ll be moved to [caskroom/versions](https://github.com/caskroom/homebrew-versions) and this one renamed. This won’t stay as `dockmod3` for sure.

Let’s do this in steps. Does Dockmod 2 work *only* with OS X versions prior to Yosemite, and does Dockmod 3 work *only* with OS X versions from Yosemite on?

**Comment by MarioStief** [comment]
Yes, thats right. The cask dockmod (DockMod 2) only works up to Mavericks. And dockmod3 only works for Yosemite (for now). Version 3 is also not backwards compatible, if you update Mavericks (which DockMod 2 can only be used on) you have a nonworking app.

**Comment by vitorgalvao** [comment]
Alright, then please check #10892 for the proper way to handle this.

Thank you for the contribution. It is still included (and still credited to you) in the mentioned PR, with the appropriate modifications. Please feel free to ask clarification about any of the changes.

**Comment by MarioStief** [comment]
I see. No need for clarification, your changes are self-explaining.
Thank you for your help. Have a nice evening. :)

**Comment by vitorgalvao** [comment]
Thank you again for the contribution.

> Have a nice evening.

Thank you, for you as well.

**Pull request closed without merging**